docs/language.md

```diff
@@ -280,3 +282,61 @@ and dynamic maps.
 * `Object.keys()` is currently not implemented for classes; when it will be
   the order of fields will be static declaration order
 * how to validate types of C++ classes (Pin mostly)?
+
+
+# Python
+
+## Supported language features
+The following language features should be fully supported and work according to the Python 3 language specification.
+
+* lists
+* dictionaries
```

Comment:
I guess we still don't do dictionary literals, but should be easy to add (I guess that's assigned to me)

Reply:
Think you'll get to this? If not, I'll remove it from the list